### PR TITLE
Replace mybloxx with supervisedenabler

### PR DESCRIPTION
**Pull request by OwOSwordsman**

### CODE REVIEW

**Comment by TheMasterOfMike** [comment]
So we’ve gotten asked to do this a lot - and basically truth be told I don’t want to do this because it bloats the guide in that SupervisedEnabler only works on 13.x+ (for some reason, unless that’s changed which I don’t think it has) which would mean we either tell people to edit a plist (which no, we’re not doing) or we tell people to use MyBloxx anyways which just makes this somewhat redundant (the amount of people who do want to do this and come from 11.3-12.x is actually higher than you think)

I have commits ready on my desktop quite honestly for SuperviseMe (a thing by nyu) but I’m just waiting for nyu to publish it onto their repository to then push said commits.

I’m not gonna close in case there’s some corrections that I need to hear or whatever but honestly I don’t want to merge this.

**Comment by OwOSwordsman** [comment]
I see, in that case, I shall close the pr.

**Comment by TheMasterOfMike** [comment]
I'm reopening this, because I just noticed the depiction now says it supports "iOS 11, 12, 13, 14"

File: docs/en_US/updating/updating-to-14.8-(blobless).md [hunk 1]
```diff
@@ -58,7 +57,7 @@ You need a pre-existing jailbreak installed to use this method.
 1. Plug your device into power and connect to the Internet with Wi-Fi
 1. Open the Settings application
 1. Tap `General` -> `Software Update`
-    - Ensure that the version you are updating to is 14.8
+    - Ensure that the version you are updating to is 14.8 **NOT 14.8.1**
```
Comment (TheMasterOfMike) [inline docs/en_US/updating/updating-to-14.8-(blobless).md:60]: The profile linked is a flat 90-day delay, it won't be 14.8.1 (and it 100% cannot be for those not on iOS versions 14.5-14.8)

Comment (OwOSwordsman) [reply docs/en_US/updating/updating-to-14.8-(blobless).md:60]: ahh okay

**Pull request merged**